tests/unit/s2n_random_test.c

```diff
@@ -644,10 +644,11 @@ static int s2n_random_test_case_without_pr_cb(struct random_test_case *test_case
 }
 
 /* Test case that turns off prediction resistance and all fork detection
- * mechanisms expect pthread_at_fork()
+ * mechanisms except pthread_at_fork()
  */
 static int s2n_random_test_case_without_pr_pthread_atfork_cb(struct random_test_case *test_case)
 {
+#if !defined (__OpenBSD__)
```

Comment:
I don't think this is sufficient testing. We should test randomness on OpenBSD, even if we can't test atfork.

Reply:
I've tightened up the `#if` to cover only `s2n_ignore_wipeonfork_and_inherit_zero_for_testing()`. 

Even if most of `s2n_random_test_case_without_pr_pthread_atfork_cb()` is preprocessed out, isn't randomness still tested in `s2n_random_test_case_without_pr_madv_wipeonfork_cb()` and `s2n_random_test_case_without_pr_map_inherit_zero_cb()`?